src/test/java/org/fundacionjala/coding/jorge/FizzBuzzTest.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ *
+ */
+public class FizzBuzzTest {
+    /**
+     *
+     */
+    private static final Object[] ARRAY_OF_2_FIZZ = {2, "Fizz"};
+    private static final Object[] ARRAY_OF_1_2 = {1, 2};
+    private static final Object[] ARRAY_OF_FIZZ_7_8_FIZZ = {"Fizz", 7, 8, "Fizz"};
+    private static final Object[] ARRAY_OF_BUZZ = {"Buzz"};
+    private static final Object[] ARRAY_OF_FIZZBUZZ = {"FizzBuzz"};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FizzBuzz" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)